README.md

```diff
@@ -97,6 +97,43 @@ Other available operations may be discovered by running:
 ./project.rb
 ```
 
+#### API: Faster API startup for MacOS
+The above steps for starting the API server can take upwards of 8-10 minutes on MacOS, most likely due to performance issues with Docker for Mac. Follow these steps to set up your developer environment to start the API server outside of docker. A full restart should take ~30 seconds with this method.
+
+All commands should be run from `workbench/api`
+
+##### Setup
```

Comment:
This implies to me a one-time setup, which is not really accurate. Here are the normal steps that happen in dev-up, and which typically may need to run during development or just when switching branches:

- Bring up services (i.e. equivalent to your `start-api-reqs`)
- Run schema migrations (pickup changes in liquibase XML files)
- Apply configuration updates (pickup changes in config/*_local.json)

Reply:
Is this resolved? I don't see any mention of how this should be handled when you're using this approach.